Sources/StreamChatUI/ChatMessageList/Attachments/ChatMessageVoiceRecordingAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,501 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatMessageVoiceRecordingAttachmentListView {
+    open class ItemView: _View, ThemeProvider {
+        // MARK: - Properties
+
+        /// Content of the attachment `ChatMessageFileAttachment`
+        public var content: ChatMessageVoiceRecordingAttachment? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        open lazy var playbackViewModel: VoiceRecordingViewPlaybackViewModel = .init(self)
```

Comment:
I'm a bit concerned about this one because we don't use view models anywhere in the current architecture of the SDK 🤔